shell/pom.xml

```diff
@@ -75,6 +75,16 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-server-common</artifactId>
```

Comment:
why does `shell` need to depend on `server`?

Reply:
Moved journal-tool to `core-server-master`.